uiowa/uiowa
# Feature: Button adjustments for home page video loop

In this PR:

- Moved play/pause button to left corner and made it larger with a solid background
- Added a media query to set `80vh` on hero container so that button remains visible (https://github.com/uiowa/uids/blob/master/src/components/hero/_hero.scss#L91-L97)
- Added aria label to play/pause button
- Mobile header adjustments to fix spacing regression documented here: https://github.com/uiowa/uids/pull/228
- Stat prefix regression fix. 

# How to test

`blt frontend`

- Test on home page video: https://uiowa.local.drupal.uiowa.edu/
- Test on interior pages with hero image.   Check breakpoints for any overflow of image into the content area such as on https://uiowa.local.drupal.uiowa.edu/admissions. 
- View source on the home page pause/play button to see that aria-label changes when paused and played. 
- Check mobile header on uiowa.local.drupal.uiowa.edu or brand.local.drupal.uiowa.edu to view that spacing is fixed https://github.com/uiowa/uids/pull/228.
- Make sure that stats on uiowa.local.drupal.uiowa.edu have prefixes displaying inline and doesn't look like:
<img width="347" alt="Screen Shot 2020-06-05 at 9 36 02 AM" src="https://user-images.githubusercontent.com/1036433/83888963-2b89fc00-a710-11ea-95d5-e95d8aee3584.png">
